### DISCUSSION TITLE
WebView2.DevTools.Dom - Version 3.01 fails...

### DISCUSSION THREAD

**Discussion by jurisquick** [ChromiumDotNet/WebView2.DevTools.Dom#13] [General]
Hello,

1. Go to Url: https://servizipst.giustizia.it/PST/it/pagopa_altripag.wp?actionPath=/ExtStr2/do/pagamentitelematici/initFormAltriPagamenti.action&currentFrame=8

2. Execute: "HighlightLinksToolStripMenuItemClick()" 

3.  WebView2.DevTools.Dom  FAIL

4. Version 2.02 however it works fine

Regards,
Lello

**Comment by amaitland** [comment]
> 3\. WebView2.DevTools.Dom  FAIL

What does `fail` mean in this context? Is there an exception, if yes then please provide a stack trace. (please no images of exceptions).

**Reply by jurisquick** [reply]
StackTrace (only this):    at A.b.E`1.A()
Hresult: -2147467261
Message: Object reference not set to an instance of an object.

**Reply by amaitland** [reply]
I'll need an `un-obfuscated` stack trace to debug this.

Signup to the [WebView2 DevTools Dom Supporter](https://github.com/sponsors/amaitland/) tier ($24USD per month) and you'll automatically gain access to the project source (here on `GitHub`). You can then run the example from the source repository and get a complete stack trace.

https://github.com/ChromiumDotNet/WebView2.DevTools.Dom#sponsorware
